src/cpp/statistics/types/monitorservice_typesTypeObjectSupport.cxx

```diff
@@ -734,7 +734,7 @@ void register_LivelinessChangedStatus_s_type_identifier(
             ReturnCode_t return_code_last_publication_handle {eprosima::fastdds::dds::RETCODE_OK};
             return_code_last_publication_handle =
                 eprosima::fastdds::dds::DomainParticipantFactory::get_instance()->type_object_registry().get_type_identifiers(
-                "anonymous_array_uint8_t_16", type_ids_last_publication_handle);
+                "anonymous_array_byte_16", type_ids_last_publication_handle);
```

Comment:
Will the changes in this file make the monitor service not to interoperate with previous versions?

Reply:
This PR only modifies the name the type is registered under. This doesn't affect the interoperability.